### PR TITLE
Change name to `collie`! 🎉 

### PR DESCRIPTION
**Pull request by nathancooperjones**
## Describe the big picture - why are we doing this?
WOOHOO - after months of back-and-forth, we finally have the `collie` name! So... let's use it! 

## Any additional details to clarify code in the PR?
No code changes here, just lots and lots of name changes! 

## How is this tested?
With unit tests of course! 

## Pull Request Checklist
 - [X] Pull request includes a description of why we are doing this
 - [X] ``CHANGELOG`` has been updated
 - [X] Version in ``_version.py`` has been updated
 - [X] All tests in the ``tests`` folder pass with a local build
 - [X] ``README`` has been updated (if applicable)
 - [X] Documentation in ``docs`` has been updated (if applicable)
 - [X] ``requirements.txt`` and ``requirements-dev.txt`` have been recompiled (if applicable)
 - [X] Docker image can be built using ``docker build -t collie .``

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,24 +20,26 @@ See the [documentation](https://collie.readthedocs.io/en/latest/index.html) for
 
 ## Installation
 ```bash
-pip install collie_recs
+pip install collie
 ```
 
+Through July 2021, this library used to be under the name ``collie_recs``. While this version is still available on PyPI, it is no longer supported or maintained. All users of the library should use ``collie`` for the latest and greatest version of the code!
```
Comment (sugi-chan) [inline README.md:26]: yeah I like having these around,

File: setup.cfg [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [flake8]
 max-line-length = 100
-application-import-names = collie_recs
+application-import-names = collie
```
Comment (sugi-chan) [inline setup.cfg:3]: yasssss

**Review by sugi-chan — APPROVED** [review]
so this looks good... but I am also kinda just frantically going through everything...

File: collie/movielens/get_data.py [hunk 3]
```diff
@@ -189,6 +189,8 @@ def read_movielens_posters_df() -> pd.DataFrame:
 
     # be prepared to read the CSV from the origin GitHub repo as well
     movielens_posters_csv_url = (
+        # 'https://raw.githubusercontent.com/ShopRunner/collie/main/data/movielens_posters.csv'
+        # TODO: remove this line once ``collie`` is merged in
         'https://raw.githubusercontent.com/ShopRunner/collie_recs/main/data/movielens_posters.csv'
```
Comment (mcundiff3) [inline collie/movielens/get_data.py:194]: Reminder to do this

**Review by peterskipper — APPROVED** [review]
Looks good! I left one question to check before you merge. I'm assuming you did a grep for collie_recs and they've all been removed?

File: tests/test_model.py [hunk 4]
```diff
@@ -605,9 +605,9 @@ def test_implicit_model(implicit_model,
                         train_val_implicit_data,
                         model_type):
     if model_type == 'with_lightning':
-        model = copy.deepcopy(implicit_model)
+        model = implicit_model
```
Comment (mcundiff3) [inline tests/test_model.py:608]: Why don't we need the deep copy anymore?

**Review by mcundiff3 — CHANGES_REQUESTED** [review]
Other than the changelog thing looks good. I just had one other questions

File: collie/movielens/get_data.py [hunk 5]
```diff
@@ -189,6 +189,8 @@ def read_movielens_posters_df() -> pd.DataFrame:
 
     # be prepared to read the CSV from the origin GitHub repo as well
     movielens_posters_csv_url = (
+        # 'https://raw.githubusercontent.com/ShopRunner/collie/main/data/movielens_posters.csv'
+        # TODO: remove this line once ``collie`` is merged in
```
[the remark this answers is not in the document]
Comment (nathancooperjones) [reply collie/movielens/get_data.py:193]: Yes it does, but I can only do it once it's merged in sadly 😞

File: CHANGELOG.md [hunk 6]
```diff
@@ -3,6 +3,14 @@ All notable changes to this project will be documented in this file.
...
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project uses [Semantic Versioning](http://semver.org/).
 
+# [1.0.0] - 2021-7-13
+### Changed
+ - name of library to ``collie``!
+
+# [0.6.1] - 2021-7-13
+### Added
+ - name change warning from ``collie_recs -> collie``
+
```
[the remark this answers is not in the document]
Comment (nathancooperjones) [reply CHANGELOG.md:13]: Yes, this PR is going to get merged in first, then this one right after: https://github.com/ShopRunner/collie_recs/pull/23

File: collie/_version.py [hunk 7]
```diff
@@ -0,0 +1 @@
+__version__ = '0.6.1'
```
[the remark this answers is not in the document]
Comment (nathancooperjones) [reply collie/_version.py:1]: Woah, good catch!!

File: tests/test_model.py (hunk 4 above)
Comment (nathancooperjones) [reply tests/test_model.py:608]: I kept getting a weird `ReferenceError: weakly-referenced object no longer exists` error for some reason all of a sudden. Removing the `deepcopy` fixes it ¯\\_(ツ)_/¯

Comment (mcundiff3) [reply tests/test_model.py:608]: So weird. But it doesn't affect the model training? Why were we using deep copy in the first place?

Comment (nathancooperjones) [reply tests/test_model.py:608]: I added it to make sure we didn't accidentally change the model fixture during any tests, but today it decided to break for no reason.

**Review by mcundiff3 — APPROVED** [review]
Looks GREAT!! GOOO COLLIE

giphy-3

**Pull request merged**